en/components/date_picker.md

```diff
@@ -125,6 +125,42 @@ Here is the formatted date:
 </div>
 <div class="divider--half"></div>
 
+#### Templating Input Group
+
+The Date Picker's input group look can be customized. To do that we need to decorate a ng-template inside the date picker with `IgxDatePickerTemplate` directive and use the context returned to customize the way the date picker's input is displayed. You can make use of `value`, `displayData` and the `openDialog`.
+
+In the following example we modify the default label "Date" add a second icon as suffix and an additional label:
```

Comment:
Replace `and an additional label` with something like `Bellow the input group we're using a label to display the real time picker value`.
The same applies to time picker as well.